mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -226,4 +324,16 @@ def save(context, name):
         Saves the current playlist to ``NAME.m3u`` in the playlist
         directory.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    tl_tracks = context.core.tracklist.get_tl_tracks().get()
+    tracks = [t.track for t in tl_tracks]
```

Comment:
You can use `context.core.tracklist.get_tracks().get()` instead to get the `Track` models directly.